Makefile

```diff
@@ -14,7 +14,7 @@ COPYRIGHT_VALUE ?= $(shell grep -i ^copyright LICENSE | sed 's/^[Cc]opyright //'
 
 OS := $(shell uname | tr '[:upper:]' '[:lower:]')
 ARCH := $(shell uname -m | if grep -q x86_64 ; then echo amd64 ; else uname -m ; fi)
-GOPATH := $(shell echo "$${GOPATH%%:*}")
+GOPATH := $(shell go env GOPATH)
```

Comment:
yeah, we should add the substitution thing back in, since the GOPATH could contain more than one value. that now also explains what the `%%:*` means, which is something we were a bit puzzled about while working on this. :)